proofread docs: for_local.md (#1565)
juicedata/juicefs: docs/en/introduction/architecture.md

![](../images/juicefs-storage-format-new.png)

> When using JuiceFS, files will eventually be split into Chunks, Slices and Blocks and stored in object storage. Therefore, you may notice that the source files stored in JuiceFS cannot be found in the file browser of the object storage platform; instead, there are only a chunks directory and a bunch of digitally numbered directories and files in the bucket. But don't panic! This is just the secret of the high-performance operation of JuiceFS!

Reviewer: ```suggestion
When using JuiceFS, files will eventually be split into Chunks, Slices and Blocks and stored in object storage. Therefore, you may notice that the source files stored in JuiceFS cannot be found in the file browser of the object storage platform; instead, there are only a directory of chunks and a bunch of numbered directories and files in the bucket. Don't panic! That's exactly what makes JuiceFS a high-performance file system.
```
1. 感觉直接 Don't panic 更 native 一些？
2. 最后一句这样表达会不会更舒服点？
